code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -8,12 +8,12 @@
 	meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/zombie
 	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOBLOOD,RADIMMUNE,NOZOMBIE,EASYDISMEMBER,EASYLIMBATTACHMENT)
 	mutant_organs = list(/obj/item/organ/tongue/zombie)
+	
 
 /datum/species/zombie/infectious
 	name = "Infectious Zombie"
 	id = "memezombies"
 	limbs_id = "zombie"
-	no_equip = list(slot_wear_mask, slot_head)

```

Comment:
I mean, you'd have to take the teleport spell, because you can't use the scroll. Unless they use the "open scroll input", then put the scroll away, change species, then finish the input selection "trick".